src/components/creatures.rs

```diff
@@ -76,8 +77,8 @@ impl Wander {
 #[serde(deny_unknown_fields)]
 pub struct CreaturePrefabData {
     pub name: Option<Named>,
-    graphics: Option<GraphicsPrefab<Vec<PosNormTex>, ObjFormat, TextureFormat>>,
-    movement: Option<Movement>,
+    graphics: Option<GraphicsPrefab<Vec<PosNormTex>>>,
```

Comment:
`PosNormTex` is not used by renderer now, as we've migrated to separate buffers in all builtin 3d passes. You have to use `GraphicsPrefab<(Vec<Position>, Vec<Normal>, Vec<TexCoord>)>`.

Reply:
Thanks for the info. This fixed the rendering of the sprites.
However, now they appear upside down. Does Rendy use a different convention for axes than the previous rendering system ?